docs/dev/table/sql/drop.md

```diff
@@ -0,0 +1,155 @@
+
+Flink SQL supports the following DROP statements for now:
+
+- DROP TABLE
+- DROP VIEW
+- DROP FUNCTION
+- DROP DATABASE
+
+## Run a DROP statement
+
+DROP statements can be executed with the `sqlUpdate()` method of the `TableEnvironment`, or executed in [SQL CLI]({{ site.baseurl }}/dev/table/sqlClient.html). The `sqlUpdate()` method returns nothing for a successful DROP operation, otherwise will throw an exception.
+
+The following examples show how to run a DROP statement in `TableEnvironment`.
+
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+EnvironmentSettings settings = EnvironmentSettings.newInstance()...
+TableEnvironment tableEnv = TableEnvironment.create(settings);
+
+// register a table named "Orders"
+tableEnv.sqlUpdate("CREATE TABLE Orders (`user` BIGINT, product STRING, amount INT) WITH (...)");
+
+// a string array: ["Orders"]
+String[] tables = tableEnv.listTable();
+
+// drop "Orders" table from catalog
+tableEnv.sqlUpdate("DROP TABLE Orders");
+
+// an empty string array
+String[] tables = tableEnv.listTable();
+{% endhighlight %}
+</div>
+
+<div data-lang="scala" markdown="1">
+{% highlight scala %}
+val settings = EnvironmentSettings.newInstance()...
+val tableEnv = TableEnvironment.create(settings)
+
+// register a table named "Orders"
+tableEnv.sqlUpdate("CREATE TABLE Orders (`user` BIGINT, product STRING, amount INT) WITH (...)");
+
+// a string array: ["Orders"]
+val tables = tableEnv.listTable()
+
+// drop "Orders" table from catalog
+tableEnv.sqlUpdate("DROP TABLE Orders")
+
+// an empty string array
+val tables = tableEnv.listTable()
+{% endhighlight %}
+</div>
+
+<div data-lang="python" markdown="1">
+{% highlight python %}
+settings = EnvironmentSettings.newInstance()...
+table_env = TableEnvironment.create(settings)
+
+# a string array: ["Orders"]
+tables = tableEnv.listTable()
+
+# drop "Orders" table from catalog
+tableEnv.sqlUpdate("DROP TABLE Orders")
+
+# an empty string array
+tables = tableEnv.listTable()
+{% endhighlight %}
+</div>
+</div>
+
+The following examples show how to run a DROP statement in SQL CLI.
+
+{% highlight sql %}
+Flink SQL> CREATE TABLE Orders (`user` BIGINT, product STRING, amount INT) WITH (...);
+[INFO] Table has been created.
+
+Flink SQL> SHOW TABLES;
+Orders
+
+Flink SQL> DROP TABLE Orders;
+[INFO] Table has been removed.
+
+Flink SQL> SHOW TABLES;
+[INFO] Result was empty.
+{% endhighlight %}
+
+
+## DROP TABLE
+
+{% highlight sql %}
+DROP TABLE [IF EXISTS] [catalog_name.][db_name.]table_name
+{% endhighlight %}
+
+Drop a table with the given table name. If the table to drop does not exist, an exception is thrown.
+
+**IF EXISTS**
+
+If the table does not exist, nothing happens.
+
+
+## DROP VIEW
+
+*TODO: should add descriptions.*
+
+## DROP FUNCTION
+
+*TODO: should add descriptions.*
```

Comment:
Please either fill these in or remove the sections entirely and re-add them in a follow up PR.